hcl/ast/merged_block.go

```diff
@@ -39,31 +43,57 @@ type MergedBlock struct {
 	RawBlocks map[string]Blocks
 }
 
+// BlockType represents a block type.
+type BlockType string
+
+// LabelBlockType represents a labelled block type.
+type LabelBlockType struct {
+	Type   BlockType // Type of the block
+	Labels string    // Labels are comma separated.
```

Comment:
```suggestion
	Labels string    // Labels are dot separated.
```

It seems that Labels are Joined by "." not ","